primitives/src/sentry.rs

```diff
@@ -335,6 +373,44 @@ pub struct Analytics {
     pub payout_count: u32,
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq)]
+#[serde(rename_all = "camelCase")]
+pub struct AnalyticsM {
```

Comment:
The `M` suffixed structs are used temporary instead of replacing all the usages of the non-M-suffixed structs